frontend/taipy-gui/src/components/Taipy/Status.spec.tsx

```diff
@@ -15,13 +15,30 @@ import React from "react";
 import {render} from "@testing-library/react";
 import "@testing-library/jest-dom";
 import userEvent from "@testing-library/user-event";
+import { CheckCircle, Error, Warning, Info } from '@mui/icons-material';
 
 import { PlusOneOutlined } from "@mui/icons-material";
 
 import Status, { StatusType } from './Status';
 
 const status: StatusType = {status: "status", message: "message"};
 
+const getStatusIcon = (status: string) => {
```

Comment:
What is the point of testing code that is not in the component ?

Reply:
Will work on it. I'm new to testing btw.

Reply:
Fair enough.
Make sure you answer your own question: "What am I testing?"
Sounds absurd, but actually drives the code pretty smoothly. And allows for a better separation of unit tests.

Reply:
Sure @FabienLelaquais

Reply:
Please check now

Reply:
I think getStatusIcon should not be duplicated here